### ISSUE TITLE
Warnings from make_wham_plots.r

### ISSUE THREAD

**Issue by brianstock** [timjmiller/wham#2]
```
There were 50 or more warnings (use warnings() to see the first 50)
> warnings()
Warning messages:
1: In par(origpar) : graphical parameter "cin" cannot be set
2: In par(origpar) : graphical parameter "cra" cannot be set
3: In par(origpar) : graphical parameter "csi" cannot be set
4: In par(origpar) : graphical parameter "cxy" cannot be set
5: In par(origpar) : graphical parameter "din" cannot be set
6: In par(origpar) : graphical parameter "page" cannot be set
```

**Comment by timjmiller** [comment]
yes, these warnings are thrown in the plotting. It doesn't cause any problems as far as I can tell, but there may be a more appropriate way of resetting par() configurations at the end of each plotting function.

**Comment by brianstock** [comment]
Fixed with commit 4ae58572e13dd18235b1895b936a93bd748fb678
